Sources/MisticaCommon/Components/GradientView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  GradientView.swift
+//
+//  Made with ❤️ by Novum
+//
+//  Copyright © Telefonica. All rights reserved.
+//
+
+import SwiftUI
+
+public struct GradientView: View {
```

Comment:
I would add doc to this class explaining why we are using this view:
- to allow dark/light mode automatically

```suggestion
/// ...
public struct GradientView: View {
```

Reply:
Doc added in GradientSwifUIViewController, it's really which is gonna use this SwiftUI view.